Libraries/Lists/__tests__/VirtualizedList-test.js

```diff
@@ -48,10 +48,22 @@ describe('VirtualizedList', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('renders empty list with empty component', () => {
+    const component = ReactTestRenderer.create(
+      <VirtualizedList
+        data={[]}
+        ListEmptyComponent={() => <empty />}
```

Comment:
Can you add headers and footers here too? I want to make sure we test the logic of whether we show them in the empty case.